src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -204,6 +205,10 @@ export class DBTCoreProjectIntegration
     );
   }
 
+  updateInitializationState(): void {
+    this.isInitialized = true;
+  }
```

Comment:
Why does this need to be interface method and can't it just be a internal thing.